cromwellApiClient/src/test/scala/cromwell/api/model/CromwellQueryResultJsonFormatterSpec.scala

```diff
@@ -11,8 +11,20 @@ class CromwellQueryResultJsonFormatterSpec extends FlatSpec with Matchers {
   behavior of "CromwellQueryResultJsonFormat"
 
   val sampleQueryResult = CromwellQueryResults(results = List(
-    CromwellQueryResult("switcheroo", WorkflowId.fromString("bee51f36-396d-4e22-8a81-33dedff66bf6"), Failed, OffsetDateTime.parse("2017-07-24T14:44:34.010-04:00"), OffsetDateTime.parse("2017-07-24T14:44:33.227-04:00")),
-    CromwellQueryResult("switcheroo", WorkflowId.fromString("0071495e-39eb-478e-bc98-8614b986c91e"), Succeeded, OffsetDateTime.parse("2017-07-24T15:06:45.940-04:00"), OffsetDateTime.parse("2017-07-24T15:04:54.372-04:00"))
+    CromwellQueryResult(
+      "switcheroo",
+      WorkflowId.fromString("bee51f36-396d-4e22-8a81-33dedff66bf6"),
+      Failed,
+      OffsetDateTime.parse("2017-07-24T14:44:34.010Z"),
+      OffsetDateTime.parse("2017-07-24T14:44:33.227Z")
+    ),
+    CromwellQueryResult(
+      "switcheroo",
+      WorkflowId.fromString("0071495e-39eb-478e-bc98-8614b986c91e"),
+      Succeeded,
+      OffsetDateTime.parse("2017-07-24T15:06:45.940Z"),
+      OffsetDateTime.parse("2017-07-24T15:04:54.372Z")
```

Comment:
Do we want to be able to to test "we can parse old formats in the database and convert them into the new format for display?"

Reply:
AFAICT Cromwell never parses old-nor-new formats from the database. JSON doesn't have a `datetime` type, so we convert everything to strings. Then when reading out metadata we convert those strings... to strings, not dates.